lib/multi-level.js

```diff
@@ -103,16 +126,18 @@ class MultiLevelCache {
 
   /**
    * Refresh the cache for a given key value pair
-   * @param  {String} key       cache key
-   * @param  {JSONSerializable} staleItem cache value
-   * @param  {UpdateFn} updateFn  async function that defines how to get a new value
-   * @param  {ShouldCache} [shouldCache] function that determines whether or not we should cache the item
+   * @param {String} key cache key
+   * @param {JSONSerializable} staleItem cache value
+   * @param {UpdateFn} updateFn async function that defines how to get a new value
+   * @param {Object} opts An options object
+   * @param {ShouldCache} [opts.shouldCache] Function that determines whether or not we should cache the item
+   * @param {Number} [opts.maxAge] The duration, in milliseconds, before a cached item expires
    *
    * @private
    * @async
    * @returns {Promise<any>} a promise that resolves once we have refreshed the correct key
    */
-  async _refresh(key, staleItem, updateFn, shouldCache) {
+  async _refresh(key, staleItem, updateFn, opts) {
```

Comment:
Seems like `opts` should have a default.
```suggestion
  async _refresh(key, staleItem, updateFn, opts = {}) {
```